openthread/cmake/extensions.cmake

```diff
@@ -0,0 +1,73 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+set(NRFXLIB_DIR ${ZEPHYR_NRFXLIB_MODULE_DIR})
+assert_exists(NRFXLIB_DIR)
+include(${NRFXLIB_DIR}/common.cmake)
```

Comment:
Not needed if this file is included from here: https://github.com/nrfconnect/sdk-nrfxlib/blob/master/openthread/CMakeLists.txt

Reply:
Thanks, fixed.